cql/ScreeningSymptomaticLibrary.cql

```diff
@@ -1,14 +1,14 @@
-/*  
```

Comment:
I'll take you up on that offer to regenerate the ELM files. I'm running into a certificate issue with my java

Reply:
I added the ELM files in this commit: https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/pull/8/commits/1bc5b9a413dfe3d6f2eeadb32566f9e4c49c0590.

Reply:
I also added one more tiny commit (https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/pull/8/commits/8d03e9d33ff58a81789cb5236be9520c08e455f0) that changes the reference to the `MostRecentNilmCytology` expression to use the one already defined in that file instead of the one from the Dashboard.